QUEST_LV_0100.tsv

```diff
@@ -119,82 +119,82 @@ QUEST_LV_0100_20150317_000118	${memo X}Isn't it better to use the cable car rath
-QUEST_LV_0100_20150317_000173	Zachariel told me to share this sad news with the qualified person.{nl}See if you are qualifed for this at the Great King's Eyes.
-QUEST_LV_0100_20150317_000174	We have one more responsiblity besides protecting this Mausoleum.{nl}To protect the sad news until the qualified person comes.{nl}
-QUEST_LV_0100_20150317_000175	I don't know what that sad news is.{nl}The king has told us to protect it so we are just following his order.
-QUEST_LV_0100_20150317_000176	Medicine Dealer
-QUEST_LV_0100_20150317_000177	Hmm... Well, there's a mint that only grows at Metos Canyon.{nl}Please bring me that.
-QUEST_LV_0100_20150317_000178	Metos Canyon is the best place to wake up your consciousness. Especially for an old man.
-QUEST_LV_0100_20150317_000179	That's right. You have some sense.{nl}
-QUEST_LV_0100_20150317_000180	Alright, it's done.{nl}Take this to Davio.
-QUEST_LV_0100_20150317_000181	Merchant Davio
-QUEST_LV_0100_20150317_000182	We brought that old man's medicine. {nl}But we spent a long time getting it. I should charge you more.{nl}
-QUEST_LV_0100_20150317_000183	{memo X}The gold that Lichenclops have in this town will be enough.{nl}Bring those to the pharmacist at Kapas.
-QUEST_LV_0100_20150317_000184	This will be enough. Davio sent you right?{nl}To make the medicine effective I need one more ingredient.
-QUEST_LV_0100_20150317_000185	To make the medicine, I needed find the cause and it drove me crazy.{nl}The funny thing is that it was the same thing as the magic that was used long ago.{nl}
-QUEST_LV_0100_20150317_000186	I heard a rumor that Zachariel erased the memories of all workers after building the Mausoleum.{nl}That's the magic cast on Jonas, though he is also a bit senile.
-QUEST_LV_0100_20150317_000187	It was so hard to get that old man's medicine.{nl}But, there's nothing that I cannot get.
-QUEST_LV_0100_20150317_000188	It will be completed when you melt it in the honey that you brought. Take it.
-QUEST_LV_0100_20150317_000189	That medicine works well. Make him swallow it somehow.
-QUEST_LV_0100_20150317_000190	It's late.{nl}Did you bring my records?
-QUEST_LV_0100_20150317_000191	Go to the Abondoned Workplace.{nl}Some old man is wandering around there. I think it's Jonas.
-QUEST_LV_0100_20150317_000192	What is that? It smells like honey alcohol.
+QUEST_LV_0100_20150317_000172	$Finally the inheritance can be passed down to someone.{nl}I regret not recognizing you sooner due to my senility.
+QUEST_LV_0100_20150317_000173	$Zachariel told us to grant his inheritance to a qualified person.{nl}You can check whether you are qualified with the Eyes of the Great King.
+QUEST_LV_0100_20150317_000174	$We have one more responsibility besides protecting this Mausoleum.{nl}And that is to protect the inheritance until a qualified person comes.{nl}
+QUEST_LV_0100_20150317_000175	$We don't know what is the inheritance.{nl}The king just told us to protect it so we are just following his order.
```

Comment:
We don't know what is the inheritance -> We have no idea what exactly the hidden treasure is. (according to SHU)